packages/components/src/components/button/button.stories.ts

```diff
@@ -69,7 +70,62 @@ export const Primary: StoryObj = {
     disabled: false,
     'soft-disabled': false,
     variant: BUTTON_VARIANTS.PRIMARY,
+    size: PILL_BUTTON_SIZES[32],
+    color: BUTTON_COLORS.DEFAULT,
+  },
+};
+
+export const PrefixIcon: StoryObj = {
+  args: {
+    children: html`<mdc-icon name="info-circle-bold" slot="prefix-icon"></mdc-icon>
+      <span>Left Icon</span>`,
+    active: false,
+    disabled: false,
+    'soft-disabled': false,
+    variant: BUTTON_VARIANTS.PRIMARY,
+    size: PILL_BUTTON_SIZES[32],
+    color: BUTTON_COLORS.DEFAULT,
+  },
+};
+
+export const PostfixIcon: StoryObj = {
+  args: {
+    children: html`<p>Right Icon</p>
```

Comment:
suggestion: a <p> tag comes with predefined styling, like padding etc which could mess up the styling of the button. I would recommend to not show this here, cause we don't want our consumers to do it that way.